ares/n64/memory/bus.hpp

```diff
@@ -20,6 +20,7 @@ inline auto Bus::read(u32 address, Thread& thread, const char *peripheral) -> u6
   if(address <= 0x1fbf'ffff) return pi.read<Size>(address, thread);
   if(address <= 0x1fcf'ffff) return si.read<Size>(address, thread);
   if(address <= 0x7fff'ffff) return pi.read<Size>(address, thread);
+  if(Model::Aleck64())       return aleck64.read<Size>(address, thread);
```

Comment:
Maybe helpful to have a freeze here too like with your burst implementations?
```suggestion
  if(Model::Aleck64()) {
    if(address <= 0xbfff'ffff) return freezeUnmapped(address), 0;
    if(address <= 0xc0800'fff) return aleck64.read<Size>(address, thread);
  }
```